packages/fxa-content-server/app/scripts/templates/settings/delete_account.mustache

```diff
@@ -14,7 +14,7 @@
     <div class="error"></div>
     {{#subscriptions.0}}
       <p class="subscription-link">
-        {{#unsafeTranslate}}Looking to cancel a paid subscription? Go to <a href="/subscriptions">Manage Subscriptions</a>.{{/unsafeTranslate}}
+        {{#unsafeTranslate}}Looking to cancel a paid subscription? Go to <a href="/subscriptions">Subscriptions & Payments</a>.{{/unsafeTranslate}}
```

Comment:
I do believe that needs to be an `&amp;` instead of plain ol' `&`

Reply:
on it, thanks